features/management_teachers/deleteTeacher.feature

```diff
@@ -1,4 +1,4 @@
-@management @managementTeachers @deleteTeacher
+@management @managementTeachers @deleteTeache r@stableTest
```

Comment:
small typo  @management @managementTeachers **@deleteTeache r@**stableTest should be @management @managementTeachers **@deleteTeacher @stable**Test